instructor-notes/solutions/fileexists_download-fastq.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -euo pipefail
+
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+# Define study ID
+STUDY_ID="SRP255885"
+
+# Define file names
+FASTQ_R1="SRR11518889_1.fastq.gz"
+FASTQ_R2="SRR11518889_2.fastq.gz"
+FASTQ_URL="ftp://ftp.sra.ebi.ac.uk/vol1/fastq/SRR115/089/SRR11518889"
+
+# Define and create destination directory for FASTQ files to live in
+FASTQ_DEST="../data/raw/fastq/$STUDY_ID/"
```

Comment:
Maybe use  {} for safety (discuss potential error with a variable named `$STUDY`)?

I'm ambivalent about removing the trailing slash; I like the indicator that it is a directory, but you add it back in every use.
```suggestion
FASTQ_DEST="../data/raw/fastq/${STUDY_ID}"
```